### PR TITLE
Bump some dependency gems to latest

### PR DESCRIPTION
**Pull request by bencrouse**

### CODE REVIEW

**Review by mttdffy — COMMENTED** [review]
Test failures seem new?

**Comment by bencrouse** [comment]
yup 😩

File: core/lib/workarea/core.rb [hunk 1]
```diff
@@ -45,8 +45,6 @@
 require 'active_merchant'
 require 'mongoid/tree'
 require 'dragonfly'
-require 'net/ftp'
```
Comment (tubbo) [inline core/lib/workarea/core.rb:48]: TIL we depended on these

File: storefront/app/controllers/workarea/storefront/analytics_controller.rb [hunk 2]
```diff
@@ -50,7 +50,7 @@ def search
       private
 
       def ignore_bots
-        if browser.bot?
+        if browser.bot.bot? || browser.bot.search_engine?
```
Comment (tubbo) [inline storefront/app/controllers/workarea/storefront/analytics_controller.rb:53]: and the award for "most useless predicate method" goes to...

**Review by tubbo — APPROVED** [review]

**Pull request merged**